app/Module.scala

```diff
@@ -13,6 +13,7 @@ class Module extends AbstractModule {
     bind(classOf[ChatsRepository]).to(classOf[SlickChatsRepository])
     bind(classOf[AuthenticationRepository]).to(classOf[SlickAuthenticationRepository])
     bind(classOf[AuthenticatedUserAction]).to(classOf[ImplAuthenticatedUserAction])
+    bind(classOf[ApplicationStart]).asEagerSingleton()
```

Comment:
I guess this will cause the app to crash if there is an exception thrown.

Be aware that "sbt run" in play starts lazily, while a production run starts eagerly. So they will behave differently in this case.